tests/app/forms/test_household_composition.py

```diff
@@ -20,7 +20,7 @@ def _error_exists(answer_id, msg, mapped_errors):
 
     def test_get_name_form_generates_name_form(self):
         with self.test_request_context():
-            form = generate_household_composition_form(self.schema, self.block_json, {})
+            form = generate_household_composition_form(self.schema, self.block_json, {}, metadata={}, group_instance=0)
```

Comment:
The function def only has positional args so technically you don't need to name `metadata` and `group_instance` keywords. Not a big issue but I'm wondering if it says something about `generate_household_composition_form` args - do we need a `*` arg list delimiter for example?

Reply:
i just named them for clarity, i guess thats a better standard but not sure myself. Raises food for thought though, all tutorials i have watched have mentioned to name args regardless

Reply:
As a rule I never name positional args and always name kwargs. TBH I've never questioned it but all python devs I've worked with have done the same.